api/filters.go

```diff
@@ -326,6 +327,46 @@ func (api *API) hashFilterDimensions(ctx context.Context, fID string) (string, e
 	return fmt.Sprintf("%x", h.Sum(nil)), nil
 }
 
+func (api *API) putFilter(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	/* sample response
```

Comment:
Is this commented code still needed?

Reply:
This is the sample response form swagger spec, left in code to illustrate what we are setting in the response

Reply:
I will remove it